author_api/models.py

```diff
@@ -30,9 +58,80 @@ def host(self):
     def url(self):
       return self.host + 'author/' + str(self.id)
 
+    objects = AuthorManager()
+
+    # mixing controller logic with model logic unfortunately
+    def add_follower(self, follower):
+        """Create a follower from an Author/CachedAuthor model"""
+        _cached = CachedAuthor.objects.get(id = follower.id)
+
+        # Prevent duplicate entries
+        if not self.followers.filter(id = follower.id):
```

Comment:
Nitpicky, but I suspect doing a get is faster here. Is there a specific reason to use filters in this instance? Lack of exception throwing?